1-JavaScript/Week3/homework/ex1-giveCompliment.js

```diff
@@ -18,13 +18,16 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/1-Java
    Use `console.log` each time to display the return value of the
    `giveCompliment` function to the console.
 -----------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+function giveCompliment(name){
+const compliments=['great','perfect','awesome','incredible','the best','magnificient','stunning','genious','smart','nice'];
+let currentIndex= Math.floor(Math.random()*10);
```

Comment:
```suggestion
let currentIndex= Math.floor(Math.random()*compliments.length);
```

You can use `compliments.length` to make this more resilient!